tcs-service/pom.xml

```diff
@@ -12,7 +12,7 @@
 
   <groupId>com.transformuk.hee.tis</groupId>
   <artifactId>tcs-service</artifactId>
-  <version>6.33.4</version>
+  <version>6.34.1</version>
```

Comment:
The same. The next minor version is 6.34.0

Reply:
Ah I added the patches because I wasn't sure the analysis would work again, will revert